src/main/java/bio/terra/workspace/service/workspace/flight/GenerateProjectIdStep.java

```diff
@@ -28,6 +28,7 @@ public StepResult undoStep(FlightContext flightContext) {
   }
 
   /** Generate a pseudo-random resource id. */
+  // TODO: Why not just use the UUID? Or use a short UUID (once I put that into common-lib)
```

Comment:
please make a ticket for short UUID support.

Reply:
I have coded up a TerraUUID class that has it in and is nicely deserialized by Stairway. So it is on the way. The ticket is PF-546